docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,145 @@
+{% from "userGuide/components/advanced.md" import slot_info_trigger %}
+
+## Annotations
+
+Annotation components allows you to easily add annotations over any image.
+
+### Introduction
+
+Annotation wrappers (`<annotate>`) are used in conjunction with annotation
+points (`<a-point>`).
+
+Annotation wrappers are used to hold the image and set the width and height of
+said image.
+
+Annotation points are used to define the position, text and style of each point
+within the image. In use, the annotation points are directly inserted between
+the annotate wrappers (`<annotate>...</annotate>`).
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="sampleImage">
+  <!-- Minimal Point -->
+  <a-point x="25%" y="25%" content="Lorem ipsum dolor sit amet" />
+  <!-- Customize Point Size (default size is 40px) -->
+  <a-point x="50%" y="25%" content="Lorem ipsum dolor sit amet"  size="60"/>
+  <!-- Customize Point Header (default is empty) -->
+  <a-point x="75%" y="25%" content="Lorem ipsum dolor sit amet"  header="Lorem ipsum"/>
+  <!-- Customize Point Color (default color is green) -->
+  <a-point x="33%" y="50%" content="Lorem ipsum dolor sit amet"  color="red"/>
+  <!-- Customize Point Opacity (default opacity is 0.3) -->
+  <a-point x="66%" y="50%" content="Lorem ipsum dolor sit amet"  opacity="0.7"/>
+  <!-- Customize Point Label (default is empty) -->
+  <a-point x="25%" y="75%" content="Lorem ipsum dolor sit amet" label="1"/>
+  <!-- Customize Text Color (default color is black) -->
+  <a-point x="50%" y="75%" content="Lorem ipsum dolor sit amet"  textColor="white" color="black" label="2" opacity="1"/>
+  <!-- Customize Font Size (default font size is 14) -->
+  <a-point x="75%" y="75%" content="Lorem ipsum dolor sit amet"  fontSize="30" label="3"/>
```

Comment:
For this part, we can consider having some meaningful annotation. For example: put an annotation in a spot where two lines cross: annotate that this is not an intersection. Another example: annotate the and gate and or gate

Reply:
That might make it harder to locate each point though. The goal here is just to demo how it works and not to show off the functionality, though I am working on a section that is focused on showing the practical aspects.

Reply:
Perhaps the content of some of the points can be the location. E.g. `<a-point x="25%" y="25%" content="point at 25% 25%" />`. With the number of points in this diagram, I was unable to figure out which relates to which line of code, especially since there's no mention of how x, y works before this.